src/magicbox/ResultPreviewsManager.ts

```diff
@@ -211,7 +211,7 @@ export class ResultPreviewsManager {
     this.resultPreviewsContainer.append(preview.element);
     const elementDom = $$(preview.element);
     elementDom.on('click', () => preview.onSelect());
-    elementDom.on('keyboardSelect', () => preview.onSelect());
+    elementDom.on('keyboardSelect', () => elementDom.el.click());
```

Comment:
Hmm, I see when there is a `click` event, we call `preview.onSelect`. Why do we now have a different handler/path for `keyboardSelect`?

Reply:
I guess the keyboardSelect triggers the click which calls selects...
That changes seems kind of weird/unnecessary to me...

Reply:
Because calling `click` from `keyboardSelect` will activate the same path along with whatever behaviour was bound to the `click` event by `QuerySuggestPreview`'s template. I was mostly trying to keep the legacy behaviour of `QuerySuggestPreview`.